File: app/src/main/java/eu/kanade/tachiyomi/data/source/model/MangasPage.java
package eu.kanade.tachiyomi.data.source.model;

import java.util.List;

import eu.kanade.tachiyomi.data.database.models.Manga;

public class MangasPage {

    public List<Manga> mangas;
    public int page;
    public String url;
    public String nextPageUrl;

    public MangasPage(int page) {
        this.page = page;
    }

}

File: app/src/main/java/eu/kanade/tachiyomi/data/source/model/MangasPage.kt
package eu.kanade.tachiyomi.data.source.model

import eu.kanade.tachiyomi.data.database.models.Manga

class MangasPage(val page: Int) {

    val mangas: MutableList<Manga> = mutableListOf()

    lateinit var url: String

    var nextPageUrl: String? = null

}
